src/utils/ga.js

```diff
@@ -0,0 +1,24 @@
+import ReactGA from 'react-ga';
+
+export function initGA() {
```

Comment:
for each of these functions (inside the exclude block), could you also add `window.location.host.endsWith('binary.com') || window.location.host.endsWith('binary.me')` to make sure the code only runs in `binary.[com|me]` domain. The `binary.sx` domains just adds noise the analytics data.